virttest/utils_net.py

```diff
@@ -1026,10 +1026,14 @@ def get_structure(self):
     def list_br(self):
         return list(self.get_structure().keys())
 
-    def list_iface(self):
+    def list_iface(self, br=None):
         """
         Return all interfaces used by bridge.
```

Comment:
Please update it since we can specify the bridge we want.

Reply:
You mean the doc string? I think "all the interfaces used by bridge" fits well. bridge could be None or specified. The function returns the interfaces used by bridge(None or brname). You think?

Reply:
Okay. Since `br` alias `bridge`, we can ignore this doc.
Acked-by: Yihuang Yu <yihyu@redhat.com>